packages/vaadin-overlay/src/vaadin-overlay.js

```diff
@@ -905,23 +905,14 @@ class OverlayElement extends ThemableMixin(DirMixin(PolymerElement)) {
     }
   }
 
-  /**
-   * @param {Element} element
-   * @return {boolean}
-   * @protected
-   */
-  _isFocused(element) {
-    return element && element.getRootNode().activeElement === element;
-  }
-
   /**
    * @param {Element[]} elements
    * @return {number}
    * @protected
    */
   _focusedIndex(elements) {
     elements = elements || this._getFocusableElements();
-    return elements.indexOf(elements.filter(this._isFocused).pop());
+    return elements.indexOf(elements.filter((element) => element && isElementFocused(element)).pop());
```

Comment:
**note:** It is the caller's responsibility to make sure that the element is not `undefined` or `null` before passing to the `isElementFocused` function.